### ISSUE TITLE
implementing tor stream isolation

### ISSUE THREAD

**Issue by ghost** [neelchauhan/stem#2]
Hi!
I want to implement tor stream isolation using stem library.
what approach is suitable for distinguishing between vast variety of streams that come to tor, by a socks user/pass basis.
i need to be notified if tor button has assigned different user/pass to that stream, so it belongs to a different tab or requested domain. by the time among  all the information i can get from a StreamEvent using stem, non of them helps me to find the user/pass associated with the new-coming stream. is there a way using stem API to do this? Thanks in Advance.

**Comment by neelchauhan** [comment]
This is not the official Stem repository. It is a development repository for a few patches I wrote a few years ago. Since then, I have refocused my Tor efforts on Tor's "core" software (the Tor daemon that communicates with the Tor network).

A better person to ask would be Damian Johnson (atagar) from the Tor Project, or on Tor's Trac here: https://trac.torproject.org/projects/tor/wiki/doc/stem/bugs
